caddytls/ctmonitor.go

```diff
@@ -132,16 +128,16 @@ func getCaddyCerts() ([]string, map[string]struct{}) {
 
 func getCertSpotterCerts(domainName string, config QueryConfig, biggestID *int, retrievedCerts map[string]CertSpotterResponse) (numOfIssuanceObjects int, issuanceID int, retryAfter string, err error) {
 	// issuanceObjects consists of the issuanceObjects returned from CertSpotter.
-	issuanceID = config.Index
 	var (
 		issuanceObjects []CertSpotterResponse
 		certQuery       string
 	)
+	issuanceID = config.Index
 	certQuery = prepQuery(domainName, config)
 
 	response, err := http.Get(certQuery)
 	if err != nil {
-		log.Printf("https get request failed on input %s \nError: %v", prepQuery(domainName, config), err)
+		log.Printf("https get request failed on input %s, Error: %v", prepQuery(domainName, config), err) // TODO fix this according to changes on git.
```

Comment:
TODO?

Consider this instead:

```
log.Printf("[ERROR] CT monitor: %v (url=%s)", err, certQuery)
```

Reply:
I wanted to make sure that I had this up to date with the requested changes on here.